interfaces/image_format.hpp

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+namespace bnb::oep::interfaces
+{
+
+    enum class rotation : int32_t
+    {
+        deg0, /* 0 degrees */
+        deg90, /* 90 degrees */
+        deg180, /* 180 degrees */
+        deg270, /* 270 degrees */
+    }; /* enum class rotation */
+
+    enum class image_format : int32_t
+    {
+        bpc8_rgb, /* one plane: RGB */
+        bpc8_bgr, /* one plane: BGR */
+        bpc8_rgba, /* one plane: RGBA */
+        bpc8_bgra, /* one plane: BGRA */
+        bpc8_argb, /* one plane: ARGB */
+        nv12_bt601_full, /* two planes: first RED, second RG */
```

Comment:
Add some notes on the difference between bt601 and bt709 - so the user can get some basic notion.